services/brig/test/integration/Util.hs

```diff
@@ -197,7 +198,10 @@ runFedClient (FedClient mgr ep) domain =
       let req' = Servant.defaultMakeClientRequest burl req
        in req'
             { HTTP.requestHeaders =
-                HTTP.requestHeaders req' <> [(originDomainHeaderName, toByteString' originDomain)]
+                HTTP.requestHeaders req'
+                  <> [ (originDomainHeaderName, toByteString' originDomain),
+                       (versionHeader, toByteString' (versionInt (maxBound :: Version)))
```

Comment:
Will this `maxBound` approach work even once we start dropping old versions?

Reply:
I think so, the assumption is that we are always testing with the maximum available version. If this assumption changes, we'll have to change this code, but I think it's fair.